app/services/tools/nkit2iso.py

```
@@ -0,0 +1,130 @@
+        split: bool = False,  # noqa: ARG002 - split applies only to makeps3iso
+        cancel_event: asyncio.Event | None = None,
+    ) -> AsyncGenerator[dict, None]:
+        return self._service.convert(
+            input_path, output_path, mode,
+            compression=compression, cancel_event=cancel_event,
+        )
+
+    async def info(self, path: str) -> dict:
+        return await run_in_threadpool(self._service.info, path)
+
+    def info_model(self, raw: dict, path: str) -> NkitInfo:
```

Ruff (0.16.1)

[warning] Unused method argument: `path`

(ARG002)